news/2021-05-07-monthly-beatmapping-contest-may-2021.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Monthly Beatmapping Contests: May 2021"
+date: 2021-05-07 08:00:00 +0000
+---
+
+Try your hand at this month's slow-slider-themed contest with a chance to become osu!'s next *Elite Mapper*!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+May's [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) (*MBC*) includes opportunities for [**osu!**](LINK), [**osu!taiko**](LINK), and [**osu!mania**](LINK) mappers. Maybe next month we'll cover every mode at the same time, who knows?
```

Comment:
```suggestion
May's [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) (MBC) includes opportunities for [**osu!**](LINK), [**osu!taiko**](LINK), and [**osu!mania**](LINK) mappers. Maybe next month we'll cover every mode at the same time, who knows?
```

Reply:
is there a new rule for formatting for these situations? ive been doing this for all the mbc news posts so far

Reply:
it doesn't matter imo. but if past newsposts have that italicised, then it would be more appropriate to leave it as is for the sake of consistency

Reply:
i don't believe so. if any this is a matter of preference i suppose